docs/reference/logging/fluentd.md

```diff
@@ -25,8 +25,16 @@ driver sends the following metadata in the structured log message:
 | `container_name` | The container name at the time it was started. If you use `docker rename` to rename a container, the new name is not reflected in the journal entries.                                         |
 | `source`         | `stdout` or `stderr`                |
 
+The `docker logs` command is not available for this logging driver.
+
 ## Usage
 
+Some options are supported by specifying `--log-opt` as many as needed:
+
+ - `fluentd-address`: specify `host:port` to connect [localhost:24224]
```

Comment:
not sure about the square brackets here, should this just be (for example, `localhost:24224`)